test/ctia/http/routes/pagination_test.clj

```diff
@@ -103,4 +124,111 @@
        (testing "judgements/indicators by observable"
          (pagination-test-no-sort app
                                   (str route-pref "/judgements/indicators")
-                                  {"Authorization" "45c1f5e3f05d0"}))))))
+                                  headers))))))
+
+(defn new-maximal-by-entity []
```

Comment:
I would add a docstring to ease the reading of the function
"returns a map of all entity maximal indexed by entity type (removing :identity and :event)."